### ISSUE TITLE
Book 05: Sentence 090

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#12]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/20574b45-c03d-4dea-8dc2-db020b831d74)

```
# sent_id = Book_05_-_The_Omens.txt:90
# text = Ic matlactli omei capitulo, vncan moteneoa: yn occequj tetzavitl, in iooaltica monextiaia.
# text[norm] = Ic matlactli omeyi capítulo, oncan motenehua: in occequi tetzahuitl, in yohualtica monextiaya.
# text[orig] = Ic·matlactli·omei·capitulo·,·vn¶can·moteneoa·:·ynoccequj·tetzavitl·,·in·iooaltica·monextiaia·.
# text[eng] = Thirteenth chapter: where are told: the other omens, that at night showed themselves.
# text[osp] = Capitulo treze, en que se trata de otras fantasmas: que aparecian de noche.
# references = A&D/179:1
1	Ic	ic	ADV	_	_	4	dep	_	Folio=12v|Paragraph=3|Line=1|Norm=Ic
2	matlactli	matlactli	NUM	_		4	nummod	_	Folio=12v|Paragraph=3|Line=1|Norm=matlactli|Gloss=diez|Analysed=Yes
3	omei	eyi	NUM	_	_	2	conj	_	Folio=12v|Paragraph=3|Line=1|Norm=omeyi|Gloss=y.tres
4	capitulo	capítulo	NOUN	_	State=Abs	7	dep	_	Folio=12v|Paragraph=3|Line=1|Norm=capítulo|Analysed=Yes
5	,	,	PUNCT	_		4	punct	_	Folio=12v|Paragraph=3|Line=1|Norm=,|Analysed=Yes
6	vncan	oncan	ADV	_		7	advmod	_	Orig=vn¶·can|Folio=12v,12v|Paragraph=3,3|Line=1,2|Norm=oncan|Gloss=aquí|Analysed=Yes
7	moteneoa	tenehua	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	Folio=12v|Paragraph=3|Line=2|Norm=motenehua|AmbigNorm=True|Override=False|Analysed=Yes
8	:	:	PUNCT	_		7	punct	_	Folio=12v|Paragraph=3|Line=2|Norm=:|Analysed=Yes
9	yn	in	DET	_	_	11	det	_	Orig=ynoccequj|Folio=12v|Paragraph=3|Line=2|Norm=in
10	occequj	occequi	DET	_	_	11	det	_	Orig=ynoccequj|Folio=12v|Paragraph=3|Line=2|Norm=occequi|Gloss=unos
11	tetzavitl	tetzahuitl	NOUN	_	State=Abs	7	csubj	_	Folio=12v|Paragraph=3|Line=2|Norm=tetzahuitl|Analysed=Yes
12	,	,	PUNCT	_		11	punct	_	Folio=12v|Paragraph=3|Line=2|Norm=,|Analysed=Yes
13	in	in	SCONJ	_	_	15	mark	_	Folio=12v|Paragraph=3|Line=3|Norm=in
14	iooaltica	yohualtica	ADV	_		15	advmod	_	Folio=12v|Paragraph=3|Line=3|Norm=yohualtica|Temporal=Yes|Analysed=Yes
15	monextiaia	nextia	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Reflexive[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	11	acl	_	Folio=12v|Paragraph=3|Line=3|Norm=monextiaya|Analysed=Yes
16	.	.	PUNCT	_		7	punct	_	Folio=12v|Paragraph=3|Line=3|Norm=.|Analysed=Yes
```

**Comment by ftyers** [comment]
Check how we want to analyse numerals like *matlactli omeyi* and cross-check analysis of ordinals.

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8e571fde-45f0-4d69-9387-5100a9eed99e)

Questions:
* What is the structure of *in occequi tetzahuitl* and what is its function?

**Comment by Lguyogiro** [comment]
>  What is the structure of in occequi tetzahuitl and what is its function?

here we can either say it is a copular clause (no _catca_ since it is in the present tense) e.g. "There are other portents (that...)" or that _tetzahuitl_ is the `nsubj` of _motenehua_ and _in_ is a `DET`. I think the clausal analysis is more consistent with what we've been doing in other cases, where there is _catca_. But I'm not sure what diagnostic we can use to distinguish these.

**Comment by ftyers** [comment]
Yeah that's tricky. There are a couple of ways to look at it:
1. No clausal analysis without the copula (or without some other indication, like an agreement prefix)
2. Clausal analysis preferred if there could potentially be a copula (e.g. we have an example of the exact same pattern with a copula)

I think I prefer (1), it's:
- easier to annotate
- parsers should be able to make the distinction
- people searching in the corpus can do `nsubj|csubj` to find any subject

The downsides are for people who want to search specifically for nominal or clausal subjects, they might not get the results they want, but in that case I think they probably need to eyeball all the examples anyway.

**Comment by ftyers** [comment]
Here is a suggestion:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/a13a698c-9f6b-431b-86df-a56767f8c552)

But I'm still not convinced of the structure of *in occequi tetzahuitl*, for one, it seems that *occequi* is almost never written together:

```
     55 oc cequi
     38 oc cequintin
      3 oc cequin
      2 occequi
```

**Comment by Lguyogiro** [comment]
> Here is a suggestion:

Looks good.

re occequi, yeah also since _oc_ can also modify other parts of speech (like verbs), it makes sense to split this as a MWT. This was discussed during the creation of the nhi treebank, but we ended up not tokenizing _oc_ in either the nhi or azz treebanks.

**Comment by ftyers** [comment]
Final:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/aae46c07-6269-475b-925c-d7cbc9bf161d)

I am dubious about `advmod(cequi, oc)` and having two `det`s on the same word.

**Comment by ftyers** [comment]
Merged in 73dc41e.
